frontend/src/features/contacts/contact/create/Organization/CreateOrganizationForm.tsx

```diff
@@ -53,21 +53,20 @@ export const CreateOrganizationForm: React.FunctionComponent = () => {
   );
 
   const formikRef = useRef<FormikProps<ICreateOrganizationForm>>(null);
-
   const onValidate = (values: ICreateOrganizationForm) => {
+    const errors = {} as any;
     try {
-      validateYupSchema(values, OrganizationValidationSchema, true, {
```

Comment:
This was previously throwing an error before the check for needsContactMethod was executed. Meaning only the validation schema could be executed and only once that was satisfied the custom validation would fire